modular_skyrat/modules/modular_items/lewd_items/code/screen_icons.dm

```diff
@@ -0,0 +1,24 @@
+//screen alert
+
+/atom/movable/screen/alert/aroused_x
+	name = "Aroused"
+	desc = "It's a little hot in here"
+	icon = 'modular_skyrat/modules/modular_items/lewd_items/icons/obj/lewd_items/lewd_icons.dmi'
+	icon_state = "arousal_small"
+	var/mutable_appearance/pain_overlay
+	var/mutable_appearance/pleasure_overlay
+	var/pain_level = "small"
+	var/pleasure_level = "small"
+
+/atom/movable/screen/alert/aroused_x/Initialize(mapload)
+	.=..()
+	pain_overlay = update_pain()
+	pleasure_overlay = update_pleasure()
+
+/atom/movable/screen/alert/aroused_x/proc/update_pain()
+	if(pain_level)
+		return mutable_appearance('modular_skyrat/modules/modular_items/lewd_items/icons/obj/lewd_items/lewd_icons.dmi', "pain_[pain_level]")
```

Comment:
There's already that in the icon variable, just use that instead, it'll be simpler.
```suggestion
		return mutable_appearance(icon, "pain_[pain_level]")
```